doc/api/stream.md

```diff
@@ -1830,6 +1830,65 @@ for await (const result of dnsResults) {
 }
 ```
 
+### `readable.forEach(fn[, options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `fn` {Function|AsyncFunction} a function to call on each item of the stream.
+  * `data` {any} a chunk of data from the stream.
+  * `options` {Object}
+    * `signal` {AbortSignal} aborted if the stream is destroyed allowing to
+      abort the `fn` call early.
+* `options` {Object}
+  * `concurrency` {number} the maximal concurrent invocation of `fn` to call
```

Comment:
maximal -> maximum? I'm not a native English speaker but I read maximal as "very much" and maximum as "at most"

Reply:
@vweevers this change makes sense in the other places that take `concurrency` like `map` and `filter` right?

(I'm not a native English speaker either so I'll happily defer :))